packages/arb-token-bridge-ui/src/hooks/useTransferDuration.ts

```diff
@@ -0,0 +1,183 @@
+import dayjs from 'dayjs'
+import { isTeleport } from '@/token-bridge-sdk/teleport'
+
+import { MergedTransaction } from '../state/app/state'
+import { useRemainingTime } from '../state/cctpState'
+import {
+  getBaseChainIdByChainId,
+  getConfirmPeriodBlocks,
+  getL1BlockTime,
+  isNetwork
+} from '../util/networks'
+
+export const DEPOSIT_TIME_MINUTES_MAINNET = 15
+export const DEPOSIT_TIME_MINUTES_TESTNET = 10
+
+export const TRANSFER_TIME_MINUTES_CCTP_MAINNET = 15
+export const TRANSFER_TIME_MINUTES_CCTP_TESTNET = 1
+
+/**
+ * TODO: An assumption should be 15 minutes for mainnet orbit deposits
+ * We should default to 15 and allow custom deposit times in orbit config (e.g. Xai should be 1 min)
+ * For now set 5 minutes for mainnet, 1 minute for testnet
+ */
+export const DEPOSIT_TIME_MINUTES_ORBIT_MAINNET = 5
+export const DEPOSIT_TIME_MINUTES_ORBIT_TESTNET = 1
```

Comment:
seeing the usage below, I think this is cleaner:

```suggestion
export const DEPOSIT_TIME_MINUTES_ORBIT = {
  mainnet: 5,
  testnet: 1
}
```

then you can access it by `DEPOSIT_TIME_MINUTES_ORBIT['testnet']` and don't need `getOrbitDepositDuration`

similarly for all other time constants